Changelog.rst

```diff
@@ -1,3 +1,15 @@
+3.0.0
+=====
+
+Please read the `migration guide <https://docs.tanker.io/latest/release-notes/identity/go/guide/>`_ before upgrading to this version if you use the provisional identity feature.
```

Comment:
This should probably link to `https://docs.tanker.io/latest/release-notes/identity/python/guide/` instead of /go/

Reply:
oops